samples/powerpoint/slide-management/get-set-slides.yaml

```diff
@@ -0,0 +1,172 @@
+order: 4
+id: powerpoint-slide-management-get-set-slides
+name: 'Get, set, load, and save slides'
+description: Get and set one or more selected slides. Load and save one or more slides.
+host: POWERPOINT
+api_set:
+    PowerPointApi: '1.5'
+script:
+    content: |-
+        $("#getSelectedSlides").click(() => tryCatch(getSelectedSlides));
```

Comment:
If the slide deck doesn't have 5 slides, the second button throws an error. My recommendation is to add a method (and a button, as the first button) to the code snippet that creates 5 slides, to ensure that the process goes smoothly.

Reply:
I added a warning that you need to have at least 5 slides.